tests/upgrade/initiate_upgrade.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -euox pipefail
+
+. common.sh
+
+# Wait for start ordering, till height 2
+bash ../networks/wait_for_chain.sh 2
```

Comment:
Why do we need waiting?
Can we write `bank send` and `create-did` before upgrade?

Reply:
Yes, we can.